### PR TITLE
test(timeutils): Add test for time Utils mixin

### PR DESCRIPTION
**Pull request by Camc314**
Adds test for timeUtils mixin

### CODE REVIEW

File: mixins/__tests__/timeUtils.spec.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Vue from 'vue';
+import timeUtils from '~/mixins/timeUtils.ts';
+
+const TestComponent = new Vue({
+  mixins: [timeUtils]
+});
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```
Comment (MrTimscampi) [inline mixins/__tests__/timeUtils.spec.ts:9]: Why? This line shouldn't be causing TypeScript issues.

File: mixins/__tests__/timeUtils.spec.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+describe('timeUtils', () => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  test('converts time from ms to ticks', () => {
+    const input = 10000;
+    const output = 1;
```
Comment (MrTimscampi) [inline mixins/__tests__/timeUtils.spec.ts:15]: It's probably nicer to put these values directly in the `expect()`.

File: mixins/__tests__/timeUtils.spec.ts (hunk 1 above)
Comment (Camc314) [reply mixins/__tests__/timeUtils.spec.ts]: Hmm it was causing me issues, I'll have another look.

Comment (Camc314) [reply mixins/__tests__/timeUtils.spec.ts]: I've added @types/jest which should fix any problems

**Review by MrTimscampi — APPROVED** [review]

**Pull request merged**